test/system/application-scans/java.spec.ts

```diff
@@ -92,16 +95,14 @@ describe("jar binaries scanning", () => {
             pluginResult = await scan({
               path: imageNameAndTag,
               "app-vulns": true,
-              [flagName]: true,
+              [flagName]: "1",
```

Comment:
why? it's a test to check the default. if you're specifying "1", then it's checking that value, but not that it's the default value necessarily.

Reply:
Reverting to `true` as I misunderstood the purpose of the boolean value